services/github/src/handlers.js

```diff
@@ -242,7 +228,7 @@ class Handlers {
     }
     let body = [
       '<details>\n',
-      '<summary>Submitting the task to Taskcluster failed. Details</summary>',
+      '<summary>Uh oh! Looks like an error! Details</summary>',
```

Comment:
Why this change?

Reply:
Because after this PR is merged it will be reporting the types of errors that do not lead to failing the submission of the task into Taskcluster. In fact, I am calling this from _final status_ handler, so _obviously_ the task had been submitted already. Instead writing another 98% similar function, I opted for changing the message by making it more generic.

Does that answer your question?

Reply:
Got it, yeah.  Would information on the stage at which the error occurred help the user (or us) later?  Maybe this function could take an extra parameter to give that information?

Reply:
I don't know. It seems to me that is obvious from the task status and the error message 🤔

Reply:
If you still convinced we should do that, I can file a bug to do that, because it seems a bit out of scope here

Reply:
Created a bug for this https://bugzilla.mozilla.org/show_bug.cgi?id=1624932